src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(double side) {
+        this.side = side;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: square, area: " + obtainTheArea() + " sq. units, side: " + side
+                + " units, color: " + getColor());
+    }
+
+    @Override
+    public double obtainTheArea() {
+        return side * side;
+    }
+}
```

Comment:
Don't begin class or method implementation with an empty line. There is an empty line at the beginning of the class definition (line 2) and before the method definitions (lines 10 and 16).